docs/components/app-screen.md

```diff
@@ -35,54 +35,15 @@ Everything considered part of the App Screen is placed into the `id=page-wrapper
 
 ## Sidebar with Navigation
 
-The first child element is the Sidebar with Navigation. This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
+The first child element of the `id=page-wrapper` is the [Sidebar with Navigation](/components/sidebar-nav/). This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
 
-- Current pages get the class `active` added to the class list of the `li` and `a` which hold the link.
+- Used in the App Screen.
+- Presents only two levels of navigation.
+- Current pages should get the class `active` added to the class list of the `li` and `a` which hold the link.
```

Comment:
This only applies to the child pages at the present moment (no indicator changes on `sidebar-item`); might want to also mention that the `sidebar-submenu` does not get revealed unless its display goes to `block`.

Reply:
that’s in a bullet item to come